### ISSUE TITLE
Estrutura do projeto inválida

### ISSUE THREAD

**Issue by rafatosta** [JvCedrazC/ProjectStore#1]
A estrutura do projeto está inválida!
**Você consegue executar algum teste da forma que está?**

Para que as classes sejam reconhecidas precisam de um pacote dentro da pasta java. 
O lado esquerdo é o seu atual projeto e no lado direito um novo projeto que criei para executar o seu antigo código.
![image](https://github.com/JvCedrazC/ProjectStore/assets/18619894/d5de6c70-2323-4895-a612-aeab28320c3e)

Desta forma não é possível a execução de nenhum método ou teste. Por favor, faça a correção e mantenha o padrão estabelecido pelo IntellJ.

**Comment by JvCedrazC** [comment]
Vou enviar o projeto corrigido

**Comment by rafatosta** [comment]
Você consegue executar alguma coisa nesse projeto?

**Comment by JvCedrazC** [comment]
Sim, consegui rodar normalmente, quando tava no padrão eu não consegui importar os métodos das outras classes

**Comment by JvCedrazC** [comment]
Professor, boa noite, Já fiz as mudanças, precisa de outro release?
